build.gradle

```diff
@@ -251,5 +251,5 @@ signing {
 tasks.withType(Test)  {
     maxParallelForks = 1
 }
-sourceSets.main.java.srcDirs += ['example/java', 'example/resources']
+sourceSets.main.java.srcDirs += ['examples/java', 'examples/resources']
```

Comment:
`examples/resources` is a resource dir, so it should be:

```groovy
sourceSets.main.resources.srcDirs += ['examples/resources']
```

BUT, if you fix this, you must also exclude `workflow.json`, `task_domain_wf.json` from the generated jar.

This doesn't matter much at the moment because no example is actually using the resources programmatically.